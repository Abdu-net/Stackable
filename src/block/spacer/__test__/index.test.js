import { blockAttributeTests } from '@stackable/test/shared'
import { settings } from '../'

describe( `${ settings.title } block`, () => {
	blockAttributeTests( { settings } )
} )
