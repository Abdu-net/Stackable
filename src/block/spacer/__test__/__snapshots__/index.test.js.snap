// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spacer block saved block HTML shouldn't change 1`] = `
<div
  className="ugb-spacer"
  style={
    Object {
      "height": "25px",
    }
  }
/>
`;

exports[`Spacer block saved default block HTML shouldn't change 1`] = `
<div
  className="ugb-spacer"
  style={
    Object {
      "height": "50px",
    }
  }
/>
`;
